src/components/createConnect.js

```diff
@@ -82,13 +86,17 @@ export default function createConnect(React) {
         constructor(props, context) {
           super(props, context);
           this.version = version;
-          this.stateProps = computeStateProps(context);
-          this.dispatchProps = computeDispatchProps(context);
+          this.store = props.store || context.store;
+
+          invariant(this.store, '`store` must be passed in via the context or props');
```

Comment:
Let's change it to

Could not find "store" in either context or props of "${this.constructor.displayName}". Either wrap the root component in a <Provider>, or explicitly pass "store" as a prop to "${this.constructor.displayName}".

and also verify that the error message is correct in a unit test.